wof/examples/production_configs/wsgi.py

```diff
@@ -1,3 +1,4 @@
-#! /home/ubuntu/miniconda/envs/wofpy/bin/python
```

Comment:
This one is trying to call a `miniconda` installation from a user named `ubuntu`  in an env named `wofpy`.
The changes another user besides @lsetiawan to have that configuration is quite slim :smile: 

PS: we really need to focus on more tests to catch these things early!